ci/php-cs-fixer.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+./tools/php-cs-fixer/vendor/bin/php-cs-fixer fix --dry-run --format=junit > /tmp/circleci-test-results/phpunit.xml
```

Comment:
You'll want to save this to a different file, e.g. `php-cs-fixer.xml`. Right now it overwrites the `phpunit` test results when it runs.

Reply:
Thanks for catching, I think I forgot to change the file name. Will change that.